react-xmasonry/build.boot

```diff
@@ -0,0 +1,31 @@
+(set-env!
+l  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]
+                  [cljsjs/react "15.4.2-2"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "1.0.2")
+(def +version+ (str +lib-version+ "-1"))
+
+(task-options!
+  pom  {:project     'cljsjs/react-xmasonry
+        :version     +version+
+        :description "React Masonry"
+        :url         "https://github.com/tsuyoshiwada/react-xmasonry"
+        :license     {"MIT" "https://opensource.org/licenses/MIT"}
+        :scm         {:url "https://github.com/cljsjs/packages"}})
+
+(deftask package []
+  (comp
+   (download :url (format "https://github.com/ZitRos/react-xmasonry/archive/master.zip")
+             :checksum "E6B128183607A37A561E58E5D913CF4C"
+             :unzip true)
+
+   (sift :move {#"^react-xmasonry-.*/dist/index.js"  "cljsjs/react-xmasonry/common/react-xmasonry.inc.js"})
+
+   (sift :include #{#"^cljsjs"})
+
+   (deps-cljs :name "cljsjs.react-xmasonry")
```

Comment:
The foreign-lib declaration in deps.cljs should require `cljsjs.react` so that React is loaded correctly before this lib: like here https://github.com/cljsjs/packages/blob/master/react-flexbox-grid/build.boot#L71

Cljs docs further document foreign lib requires: https://clojurescript.org/reference/packaging-foreign-deps